TFT/src/User/Menu/Babystep.c

```diff
@@ -25,14 +28,17 @@ void babyReDraw(float babystep, float z_offset, bool force_z_offset, bool skip_h
   setFontSize(FONT_SIZE_LARGE);
   sprintf(tempstr, "% 6.2f", babystep);
   GUI_DispStringRight(point_bs.x, point_bs.y, (uint8_t*) tempstr);
-  sprintf(tempstr, "% 6.2f", z_offset);
+  if (infoMachineSettings.firmwareType != FW_REPRAPFW)
```

Comment:
please add an empty line before the `if`.

Reply:
fixed